jax/core.py

```diff
@@ -317,6 +318,19 @@ def lift(self, tracer):
   def sublift(self, tracer):
     assert False
 
+  def process_primitive(self, primitive, tracers, params):
+    """Processes a primitive
+
+    Args:
+      primitive: the primitive
```

Comment:
I think a description like "Process a primitive" doesn't add anything on top of the method name `process_primitive`. Similarly parameter descriptions like "primitive: the primitive", etc don't add much. Including low-information docstrings like this is just adding noise.

I think this docstring should either be improved (e.g. to mention the types of the arguments, like "primitive: a core.Primitive instance", "tracers: a list or tuple of core.Tracer instances, one for each value-level argument to the primitive, passed as positional arguments to Primitive.bind", "params: a dict of parameters, passed as keyword arguments to Primitive.bind", etc, though if we're after type info then type annotations might work better) or deleted. I lean to deletion.

Reply:
I'll drop it for now. I do want to have a way to document what a Trace class does, and docstrings seem like a good place. I will add type annotations next. Perhaps in many cases between the type annotations and maybe explicit names for parameters it can become clear what is going on (for Trace and Tracer, I do not think that this will be sufficient).